src/views/Join/index.js

```diff
@@ -27,9 +27,15 @@ class Join extends Component {
   componentDidMount() {
     const { partyId } = this.state;
     const { navigation } = this.props;
+    let id = partyId;
+
+    if (navigation.state.params) {
```

Comment:
What are we doing here?

Reply:
Before this change, the app crashes when getting into the component without the `partyId` param (can't get the partyId from null). So, what we're doing here is setting the id from the state initially (default value) and in case of param received handle it.

Reply:
@Randyjp It seems to work but, in case we don't receive any params from the navigation the app crashes when trying to read partyId from null.